atom/browser/api/atom_api_window.cc

```diff
@@ -124,6 +124,14 @@ void Window::OnWindowRestore() {
   Emit("restore");
 }
 
+void Window::OnWindowResize() {
+  Emit("resize", GetBounds().size());
```

Comment:
Won't this cause us to ask for Window position / size a lot? I'd rather just emit the event and let the app ask for the window position / size if they need it